lisa/sut_orchestrator/azure/tools.py

```diff
@@ -55,3 +57,51 @@ def get_generation(self) -> str:
         else:
             generation = "1"
         return generation
+
+
+class LisDriver(Tool):
+    """
+    This is a virtual tool to detect/install LIS drivers.
+    """
+
+    @property
+    def command(self) -> str:
+        return "modinfo hv_vmbus"
+
+    def _check_exists(self) -> bool:
+        if not self.node.os.is_posix:
+            return False
+
+        if isinstance(self.node.os, Fedora):
+            # currently LIS is only supported with Fedora
+            # and its derived distros
+            if self.node.os.package_exists(
+                "kmod-microsoft-hyper-v"
+            ) and self.node.os.package_exists("microsoft-hyper-v"):
+                return True
+        return False
```

Comment:
It's ok, that you implement Fedora here, but raise exception for other distros, so that people know where to fix it.